RedpandaStats/README.md

```diff
@@ -0,0 +1,60 @@
+# RedPanda and Deephaven
```

Comment:
general question - do we want to develop this into a longer article, posted on deephaven.io under Examples, and keep what is in examples just code and data? Or leave this as is to support an Example article / blog post